backend/src/main/java/com/exadel/carinsurance/utils/NoOpPasswordEncoder.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.carinsurance.utils;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+public class NoOpPasswordEncoder implements PasswordEncoder {
+  @Override
+  public String encode( CharSequence rawPassword ) {
+    return rawPassword.toString();

```

Comment:
as you use real password encoder under the hood, consider delegating to it's encoder as well

Reply:
Done